### PR TITLE
feat: User activity global search

### PR DESCRIPTION
**Pull request by rephus**
Closes:  #4262

### Description:

[Screencast from 01-02-23 08:57:12.webm](https://user-images.githubusercontent.com/1983672/215986027-2a8a203a-c243-43bd-9336-0cc8d3d5f394.webm)

### CODE REVIEW

File: packages/backend/src/services/SearchService/SearchService.ts [hunk 1]
```diff
@@ -82,6 +82,14 @@ export class SearchService {
...
             savedCharts: results.savedCharts.filter(filterItem),
             spaces: results.spaces.filter(filterItem),
+            pages: user.ability.can(
+                'view',
+                subject('Analytics', {
+                    organizationUuid,
+                }),
+            )
+                ? results.pages
+                : [], // For now there is only 1 page and it is for admins only
```
Comment (rephus) [inline packages/backend/src/services/SearchService/SearchService.ts:92]: We only need to show this page to admins (and a future dashboard analytics) , if we add another page for other users we'll have to think how to handle permissions.

**Comment by owlas** [comment]
@rephus do you know why the `/dbt` sub repo was updated and the `events.yml` changed?

File: packages/common/src/types/search.ts [hunk 2]
```diff
@@ -55,6 +62,7 @@ export type SearchResults = {
     savedCharts: SavedChartSearchResult[];
     tables: TableSearchResult[];
     fields: FieldSearchResult[];
+    pages: PageResult[];
```
Comment (owlas) [inline packages/common/src/types/search.ts:65]: Cool we can add more pages like the settings page in future!

**Review by owlas — CHANGES_REQUESTED** [review]
Looks good! Just wondering what change has happened in the `events.yml` ?

**Comment by rephus** [comment]
Yes sorry, good catch. I had some local changes on dbt folder, I reverted it back.

**Review by owlas — APPROVED** [review]
🐑 it

**Pull request merged**